JoinColony/colonyCDapp
# Port "Add Safe" step 1 dialog

Safes to test the input:
`0x3F107AFF0342Cfb5519A68B3241565F6FC9BAC1e` (Ethereum)
`0xe759f388c97674D5B8a60406b254aB59f194163d` (Binance Smart Chain)

### Testing time!
- Check that the instructions link works as expected.
- Check that you can access the dialog only if you have the root permission.
- Check that the contract address input shows the correct status depending on whether the safe belongs in the chain or not.

### Known bugs
- When you change from one chain to another with a valid safe address in the input, the status message `Safe found on X chain` will appear for some milliseconds before being replaced by the correct "This safe doesn't exist in this chain". Unfortunately, I haven't been able to fix this after trying several approaches, if you guys have any ideas then they are 100% welcome.

![FireShot Capture 086 - Colony CDapp - localhost](https://github.com/JoinColony/colonyCDapp/assets/18473896/d056e236-8ef0-436e-bb90-800ed3c1d2a3)
![FireShot Capture 087 - Colony CDapp - localhost](https://github.com/JoinColony/colonyCDapp/assets/18473896/2e0f7e6d-5969-426d-af04-0f9f9ce80104)

Resolves #685
